section2/exercises/if-statements.js

```diff
@@ -44,6 +44,13 @@ Log to the console the appropriate age range
 var dogAge = 3;
 // Write your conditional here
 
+if (dogAge <= 2) {
+  console.log("puppy");
+} else if (dogAge > 2) {
```

Comment:
If the dog is 12, it would hit this `else if` and evaluate to TRUE, which you probably don't want - since we have that final `else if` - want to re-work this?